packages/expo-cli/src/commands/init.ts

```diff
@@ -161,53 +163,194 @@ async function action(projectDir: string, command: Command) {
     packageManager = 'npm';
   } else if (await shouldUseYarnAsync()) {
```

Comment:
Unrelated: Maybe we should move this into package manager like we did with `isUsingYarn`